querydsl-codegen/src/main/java/com/querydsl/codegen/EntitySerializer.java

```diff
@@ -620,6 +663,10 @@ public void serialize(EntityType model, SerializerConfig config,
 
     protected void serialize(EntityType model, Property field, Type type, CodeWriter writer,
             String factoryMethod, String... args) throws IOException {
+
+        String constantFieldName = constantFieldName(field.getEscapedName());
+        writer.publicStaticFinal(new ClassType(String.class), constantFieldName, "\"" + field.getName() + "\"");
```

Comment:
@oleg-smith,
Probably option to toggle this feature could be added as @Named constructor param like keywords param in EntitySerializer constructor.
`@Named("keywords") Collection<String> keywords`

Then default value for it could be binded in CodegenModule.java:69
`bind(KEYWORDS, Collections.<String>emptySet());`

 And in DefaultConfiguration.java you could rebind it to a value gotten from options map.
```
if (options.containsKey(QUERYDSL_PACKAGE_SUFFIX)) {
            module.bind(CodegenModule.PACKAGE_SUFFIX, Strings.nullToEmpty(options.get(QUERYDSL_PACKAGE_SUFFIX)));
        }
```